src/sympc/tensor/share_tensor.py

```diff
@@ -439,3 +439,65 @@ def method(
     __rmatmul__ = rmatmul
     __truediv__ = div
     __xor__ = xor
+
+
+allowed_external_api_share_tensor = [
```

Comment:
This needs to be split, right?

Reply:
Maybe we can put those in a folder named `api` and then have 3 lists like:
`
sympc.api.allowed_external_classes
sympc.api.allowed_external_attrs
sympc.api.allowed_external_modules
```
Approximately the same name you have in the Syft module